### DISCUSSION TITLE
New scheduling and Tesla!ate

### DISCUSSION THREAD

**Discussion by Olyts** [teslamate-org/teslamate#4181] [General]
Hi, since Tesla made some changes to charge and preconditioning scheduling, TeslaMate won’t report the charge start time as it was before the new scheduling change by Tesla. Let’s say I planned a daily charge to be completed @ 6 am, TeslaMate does not show at what time the charge will start. This is useful when you are on a low cost during night to better plan to time range to charge the car. Is it possible to bring it back?

**Comment by cwanja** [comment]
It is being discussed in #4129

**Reply by djpalmis** [reply]
Anyway, in my opinion, new scheduler is completely bugged. Even official Tesla app does not work since new firmware. https://teslamotorsclub.com/tmc/threads/charging-schedule-fails-fw-2024-26-3-1.331769/
